src/main/resources/edu/cornell/kfs/module/purap/document/validation/configuration/PurchaseOrderAmendmentValidation.xml

```diff
@@ -109,4 +109,53 @@
   			</list>
   		</property>
     </bean>
+
+    <bean id="PurchaseOrderAmendment-addAccountingLineDefault-failFastValidation" abstract="true" parent="CompositeValidation" scope="prototype">
```

Comment:
I cannot find what changed here compared to baseline?

Reply:
I changed one of the sub-bean references from "Purchasing-accountingLineDataDictionaryValidation" to the new "PurchaseOrderAmendment-accountingLineDataDictionaryValidation" bean in this PR. Let me know if you think I need XML comments to mark that change.

Reply:
no need to update. Thank you for pointing that out!